open-company/open-company-web
# Profile component

card: https://trello.com/c/7qt2sf9o

To test:
- visit [http://localhost:3449/companies](http://localhost:3449/companies) (auth with slack if you need)
- check that your avatar and your name is in the upper right corner
- click it to visit the user profile page: [http://localhost:3449/profile](http://localhost:3449/profile)
- check that your data are displayed
- go back 1 step in your browser's history ([http://localhost:3449/companies](http://localhost:3449/companies))
- select a company
- check the upper left corner, there should be a logo with the first letter of the company name
